java/code/src/com/redhat/rhn/manager/audit/CVEAuditManager.java

```diff
@@ -70,6 +70,8 @@
  */
 public class CVEAuditManager {
 
+    private static final Pattern LIVEPATCH_PACKAGES_REGEX =
```

Comment:
Nitpick: Same as my other comment. I would suggest define it in method where it is being used unless it being used in multiple places.